content/en/docs/intro/using_helm.md

```diff
@@ -263,7 +263,7 @@ There are two ways to pass configuration data during install:
 - `--set`: Specify overrides on the command line.
 
 If both are used, `--set` values are merged into `--values` with higher
-precedence. Overrides specified with `--set` are persisted in a ConfigMap.
+precedence. Overrides specified with `--set` are persisted in a Secret.
```

Comment:
Is this true? A Secret is a type of ConfigMap, no?

Reply:
They're separate API resource types. Helm v2 used ConfigMaps; Helm v3 uses Secrets.